components/wso2is.key.manager.userinfo.endpoint/src/gen/java/org/wso2/is/key/manager/userinfo/endpoint/ClaimsApi.java

```diff
@@ -0,0 +1,64 @@
+package org.wso2.is.key.manager.userinfo.endpoint;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.apache.cxf.jaxrs.ext.MessageContext;
+import org.wso2.is.key.manager.userinfo.endpoint.dto.ClaimListDTO;
+import org.wso2.is.key.manager.userinfo.endpoint.dto.ClaimRequestDTO;
+import org.wso2.is.key.manager.userinfo.endpoint.dto.ErrorDTO;
+import org.wso2.is.key.manager.userinfo.endpoint.impl.ClaimsApiServiceImpl;
+
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+@Path("/claims")
+
+@Api(description = "the claims API")
+@Consumes({ "application/json" })
+@Produces({ "application/json" })
+
+
+public class ClaimsApi  {
+
+  @Context MessageContext securityContext;
+
+ClaimsApiService delegate = new ClaimsApiServiceImpl();
+
+
+    @POST
+    @Path("/generate")
+    @Consumes({ "application/json" })
+    @Produces({ "application/json" })
+    @ApiOperation(value = "Register user ", notes = "This API is used to get user claims. ", response = ClaimListDTO.class, tags={ "User Claims",  })
```

Comment:
what does `Register user` mean?

Reply:
seems like there is an issue in the summary section in the yaml file that we have to correct. https://github.com/wso2-extensions/apim-km-wso2is/blob/master/components/wso2is.key.manager.userinfo.endpoint/src/main/resources/userinfo.yaml#L57